v1/tarball/write.go

```diff
@@ -29,5 +32,89 @@ type WriteOptions struct {
 
 // Write saves the image as the given tag in a tarball at the given path.
 func Write(p string, tag name.Tag, img v1.Image, wo WriteOptions) error {
```

Comment:
nit: Should `Write` take a `*WriteOptions` instead, so callers can pass `nil` for "default options"?

Reply:
Probably. @mattmoor did you have any reason for making this a struct instead of a pointer initially?